### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by talSofer**

### CODE REVIEW

File: deployments/compose/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # lakeFS `docker-compose`
 
-Compose file for lakeFS using variety of services lakeFS can work with.
+Compose file for lakeFS using variety of services lakeFS can work with.
```
Comment (guy-har) [inline deployments/compose/README.md:3]:
```suggestion
Compose file for lakeFS using variety of services lakeFS can work with.
```

Comment (nopcoder) [inline deployments/compose/README.md:3]: can remove the extra space

File: deployments/compose/README.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ That's it! To check the status of the containers:
 docker compose ps
 ```
 
+To access the lakeFS UI, go to http://localhost:8000. The setup stage is done for you, and your user name and passwords are AKIAIOSFODNN7EXAMPLE and wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY compatibly.
```
Comment (guy-har) [inline deployments/compose/README.md:16]:
```suggestion
To access the lakeFS UI, go to http://localhost:8000. 
Your credentials are (The setup stage was done for you):
user:AKIAIOSFODNN7EXAMPLE
pass: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY 
```

**Review by guy-har — APPROVED** [review]
Approved, my comments are only suggestions.

File: deployments/compose/README.md (hunk 2 above)
Comment (nopcoder) [inline deployments/compose/README.md:16]:
```suggestion
To access the lakeFS UI, go to http://localhost:8000. The docker-compose also runs the setup step and configure admin user with the following credentials:

Access key ID: `AKIAIOSFODNN7EXAMPLE`
Secret access key: `wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY` 
```

Comment (nopcoder) [reply deployments/compose/README.md:16]: chips

**Review by nopcoder — CHANGES_REQUESTED** [review]

File: deployments/compose/README.md (hunk 1 above)
Comment (talSofer) [reply deployments/compose/README.md]: done

File: deployments/compose/README.md (hunk 2 above)
Comment (talSofer) [reply deployments/compose/README.md]: done

Comment (nopcoder) [reply deployments/compose/README.md]: UI uses the terms 'access key id' and 'secret access key' like S3.

**Review by nopcoder — APPROVED** [review]

File: deployments/compose/README.md (hunk 2 above)
Comment (talSofer) [reply deployments/compose/README.md]: thanks, fixed.

**Pull request merged**